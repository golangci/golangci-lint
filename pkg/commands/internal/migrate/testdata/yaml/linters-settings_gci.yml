issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  gci:
    sections:
      - standard
      - default
      - prefix(github.com/org/project)
      - blank
      - dot
      - alias
      - localmodule
    no-inline-comments: true
    no-prefix-comments: true
    skip-generated: false
    custom-order: true
    no-lex-order: true
