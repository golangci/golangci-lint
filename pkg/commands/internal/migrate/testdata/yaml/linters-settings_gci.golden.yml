version: "2"
formatters:
  settings:
    gci:
      sections:
        - standard
        - default
        - prefix(github.com/org/project)
        - blank
        - dot
        - alias
        - localmodule
      no-inline-comments: true
      no-prefix-comments: true
      custom-order: true
      no-lex-order: true
